python/tests/test_almanac.py

```diff
@@ -16,3 +17,32 @@ def test_imports():
 
   """
   assert True
+
+def test_init():
```

Comment:
Great.

Would not mind seeing a quick sanity test for the calc functions, which I think you can get from an ephemeris definition?